docs/apache-airflow-providers-apache-beam/commits.rst

```diff
@@ -28,14 +28,26 @@ For high-level changelog, see :doc:`package information including changelog <ind
 
 
 
+4.1.1
+.....
+
+Latest change: 2023-01-03
+
+=================================================================================================  ===========  =================================================================================
+Commit                                                                                             Committed    Subject
+=================================================================================================  ===========  =================================================================================
+`8da678ccd2 <https://github.com/apache/airflow/commit/8da678ccd2e5a30f9c2d22c7526b7a238c185d2f>`_  2023-01-03   ``Ensure Beam Go file downloaded from GCS still exists when referenced (#28664)``
+=================================================================================================  ===========  =================================================================================
+
 4.1.0
 .....
 
-Latest change: 2022-11-11
+Latest change: 2022-11-15
 
 =================================================================================================  ===========  ====================================================================================
 Commit                                                                                             Committed    Subject
 =================================================================================================  ===========  ====================================================================================
+`12c3c39d1a <https://github.com/apache/airflow/commit/12c3c39d1a816c99c626fe4c650e88cf7b1cc1bc>`_  2022-11-15   ``pRepare docs for November 2022 wave of Providers (#27613)``
```

Comment:
```suggestion
`12c3c39d1a <https://github.com/apache/airflow/commit/12c3c39d1a816c99c626fe4c650e88cf7b1cc1bc>`_  2022-11-15   ``Prepare docs for November 2022 wave of Providers (#27613)``
```
Worth fixing this typo (and in a few other places)?

Reply:
Commits are automatically generated and overwritten each time docs are generated. So it's not really good idea to fix typos there.

Reply:
Right on, makes sense. Didn't realize they were all overwritten each time. Thanks for the context.